src/utils/hasTouchHandler.tsx

```diff
@@ -0,0 +1,25 @@
+import type { GestureResponderEvent } from 'react-native';
+
+type TouchableEventObject = {
+  onPress?: ((event: GestureResponderEvent) => void) | undefined;
+  onPressIn?: ((event: GestureResponderEvent) => void) | undefined;
+  onPressOut?: ((event: GestureResponderEvent) => void) | undefined;
+  onLongPress?: ((event: GestureResponderEvent) => void) | undefined;
+};
```

Comment:
Move the `touchableEvents` array out of the function and mark it as const and then update the type:
```suggestion
type TouchableEventObject = Partial<Record<keyof touchableEvents[number], (event: GestureResponderEvent) => void>>;
```
<img width="632" alt="image" src="https://user-images.githubusercontent.com/8790386/230576288-3d05be25-1818-4e3c-a564-c3df16ad62c0.png">